dart/example/monster.fbs

```diff
@@ -0,0 +1,33 @@
+// Example IDL file for our monster's schema.
```

Comment:
for example, this files can't be copied, regardless of what structure you choose for the other files.